cachito/workers/pkg_managers/gomod.py

```diff
@@ -187,7 +189,47 @@ def resolve_gomod(app_source_path, request, dep_replacements=None):
             )
             shutil.copytree(tmp_download_cache_dir, str(bundle_dir.gomod_download_dir))
 
-        return module, deps
+        # package level dependencies
+        go_list_deps_output = run_gomod_cmd(
+            ("go", "list", "-deps", "-f", "{{if not .Standard}}{{.ImportPath}} {{.Module}}{{end}}"),
+            run_params,
+        )
+
+        pkg_level_deps = []
+        for line in go_list_deps_output.splitlines():
+            # line example: pkg.io/foo/bar pkg.io/foo/var v1.0.0 => pkg.io/foo/bar v1.0.1
+            parts = [part for part in line.split(" ") if part not in ("")]
```

Comment:
This would avoid adding an empty part in case there would be no module info for a package. This case cannot be reached though (at first I thought we would need an additional field that could be null, where this case could be true, hence the reason for this statement). I will remove the statement.